src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.html

```diff
@@ -117,6 +117,9 @@ <h3 class="bottom_comment_payment">{{ 'order-form.payment' | translate }}</h3>
     <button class="secondary-global-button btn cansel-button btn-main" (click)="cancelUBS()">
       {{ 'order-details.cancel' | translate }}
     </button>
-    <button (click)="redirectToOrder()" type="submit" class="primary-global-button btn">{{ 'submit-order.to-order' | translate }}</button>
+    <button (click)="redirectToOrder()" type="submit" class="primary-global-button btn">
+      <span *ngIf="loadingAnim" class="spinner-border spinner-border-sm" role="status" aria-hidden="true"> </span>
```

Comment:
why do we need an empty span ?

Reply:
Forgot to remove it.

Reply:
let's use spinner from shared folder

Reply:
We need there a spinner for a button, in shared folder we have spinner for page loader
My solution with span:
![image](https://user-images.githubusercontent.com/43093994/123656903-fd0a2200-d838-11eb-9c4a-7ba0b1655398.png)